tests/api/resource_test.py

```diff
@@ -4,6 +4,8 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+from functools import wraps
```

Comment:
don't think you need this?

Reply:
maybe i don't, mock.patch was making me jump through some hoops, let me check.